vyper/old_codegen/parser_utils.py

```diff
@@ -388,105 +387,51 @@ def unwrap_location(orig):
 # Pack function arguments for a call
 @type_check_wrapper
 def pack_arguments(signature, args, context, stmt_expr, is_external_call):
+    # FLAG cyclic dep
```

Comment:
These show up in most text editors in a different color
```suggestion
    # TODO: cyclic dep
```

Reply:
I didn't make it a TODO because I don't know what our policy is on cyclic dependencies. I just added a keyword FLAG so that when we do decide we will know where to find it.

Reply:
Use `NOTE` then, that shows up in most editors as well. `FLAG` does not

Reply:
done